internal/circonus/check.go

```diff
@@ -76,6 +79,13 @@ func NewCheck(parentLogger zerolog.Logger, cfg *config.Circonus, clusterCfg *con
 		log:         parentLogger.With().Str("pkg", "circonus.check").Logger(),
 	}
 
+	var err error
+	c.clusterVers, err = k8s.GetVersion(clusterCfg)
+	if err != nil {
+		c.log.Warn().Msg("couldn't get k8s version")
+		return nil, err
```

Comment:
Does this error need to be wrapped to give it context for the user, or is the error from k8s.GetVersion clear enough as it is? Also, if this is going to fail the NewCheck() function and return an error, does the log entry generated above need to be an error instead of a warning, and should it include the error in it i.e. `.Err(err)`?

Reply:
addressed in 8b8935a